test/e2e/delegation.js

```diff
@@ -153,18 +161,33 @@ test(`delegation`, async function(t) {
     t.end()
   })
 
-  // TODO uncomment when redelegation tab is added again
-  // t.test(`Parameters`, async function(t) {
-  //   // Select the Parameters tab.
-  //   await app.client.click(`//a[. = 'Parameters']`)
-  //
-  //   await t.notOk(
-  //     await app.client.waitForExist(`.tm-notification`, 2 * 1000),
-  //     `should not get an notification error while fetching params and pool`
-  //   )
-  //
-  //   t.end()
-  // })
+  t.test(`Parameters`, async function(t) {
+    await navigate(app, `Staking`)
+    await app.client.click(`//a[. = 'Parameters']`)
+    await t.ok(
+      await app.client.waitForVisible(
+        `//h3[contains(text(), "Staking Pool")]`,
+        1000
+      ),
+      `Shows staking pool`
+    )
+    await t.ok(
+      await app.client.waitForVisible(
+        `//h3[contains(text(), "Staking Parameters")]`,
+        1000
+      ),
+      `Shows staking parameters`
+    )
+    await t.ok(
+      !(await app.client.isExisting(`//dd[contains(text(), "n/a")]`)),
```

Comment:
Maybe add a positive test. So testing that a certain value is there

Reply:
added